en_us.lang

```diff
@@ -288,11 +288,11 @@ gui.xaero_waypoint_distance_visibility_angle=WP Dist. Horis. Angle
 gui.xaero_waypoint_distance_vertical_visibility_angle=WP Dist. Vertic. Angle
 gui.xaero_teleportation=Teleportation
 gui.xaero_teleportation_tooltip=Once disabled, it can be enabled back only through the config file!
-gui.xaero_use_multiworld_tooltip=Disable if this is a simple server with a single world (no lobbies, separate worlds for different minigames, etc.). Multiworld detection can cause only issues on such servers. However, installing this mod on the server side should prevent these issues.
+gui.xaero_use_multiworld_tooltip=Disable if this is a simple server with a single world (no separate lobby, game mode, or minigame worlds). Multiworld detection may cause only issues on such servers. However, installing this mod on the server side should prevent these issues.
```

Comment:
Issues is the only thing it can cause, so I think the word "can" worked better!

Reply:
I'm not sure if "may" conveys the same meaning. Maybe "will" is the best option tbh.

Reply:
After your argumentation, I think that "can" was fine here. Should I rather use "will", or maybe go back to "can"?

Reply:
Let's go with "will"

Reply:
Done.